Events.php

```diff
@@ -47,6 +47,7 @@ public static function onRestApiAddRules()
         $restModule->addRules([
             ['pattern' => 'auth/register/', 'route' => 'smartVillage/user/user/create', 'verb' => 'POST'],
             ['pattern' => 'space/<spaceId:\d+>/membership/<userId:\d+>/request', 'route' => 'smartVillage/user/membership/request-for-membership', 'verb' => 'POST'],
+            ['pattern' => 'mail/show-messages', 'route' => 'smartVillage/mail/message/index', 'verb' => 'GET'],
```

Comment:
is this equal to `v1/mail` but with the addition of our two new fields? couldn't it be a `v2/mail` instead of a completely new naming?

Reply:
Hi @donni106 
Yes, we can change the v1 to v2 but we have to changes our previous endpoints as well.

Reply:
ah ok, i understand. so we could either have everything as `v2` with `v2/mail` or keep `v1` but then we need to have a new `mail/show-messages` because `v1/mail` already exists, right? then i would change everything to `v2`. or what do you  think would be the better approach?

Reply:
yes, we can change name from v1 to v2. currently we are using addRule function of rest humhub module in our smartVillageApi Module. So, if we want to use v2 in our module we have to write addRule function in our module.php file.

Thanks!

Reply:
Hi @donni106 
We changed API prefix url from v1 to v2.
Thanks!